tablo/models.py

```diff
@@ -432,25 +451,32 @@ def _validate_where_clause(self, where):
         parsed = self._parse_where_clause(where)
 
         if parsed is None:
-            return True
+            return
         elif parsed[1]:
-            return False  # More than one statement means SQL injection.
-        else:
-            return self._validate_fields(parsed[0])
+            raise SQLInjectionError('Invalid where clause')
```

Comment:
I guess I'm of the opinion that we shouldn't introduce a new error type unless we're going to change behavior based on that type.  Since, in this case, we know the issue is the where clause, couldn't we just raise an `InvalidFieldsError('Invalid where clause')` so that that gets reported out to the user? Unless you meant something different with the InvalidFieldsError.